extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.css

```diff
@@ -232,7 +232,8 @@ amp-story-shopping-attachment a {
   border-radius: 0 1.1em 1.1em 0 !important;
 }
 
-.i-amphtml-amp-story-shopping-pdp-carousel-card:only-child {
+.i-amphtml-amp-story-shopping-pdp-carousel-card:only-child,
+[dir=rtl] .i-amphtml-amp-story-shopping-pdp-carousel-card:only-child {
```

Comment:
It looks like the preexisting selector should already be selecting the elements of your new, more restrictive selector. Out of curiosity, why was the preexisting selector not working for the RTL card?

Reply:
I believe it's due to these selectors having a higher specificity since they have they also include the RTL selector:
https://github.com/processprocess/amphtml/blob/main/extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.css#L247-L253

Reply:
That made me realize this block needs to be after those selectors. 
Thanks for pointing this out Corey :)

Reply:
Got it, thanks!

Reply:
Actually I can remove the rtl selector and get specificity by using `:first-child:last-child`.
It's one less line but does require a comment since it's less readable than :only-child.

Reply:
Sounds good to me